src/hotspot/share/classfile/moduleEntry.cpp

```diff
@@ -272,7 +273,8 @@ ModuleEntry::ModuleEntry(Handle module_handle,
     _has_default_read_edges(false),
     _must_walk_reads(false),
     _is_open(is_open),
-    _is_patched(false) {
+    _is_patched(false),
+    DEBUG_ONLY(_reads_is_archived(false) COMMA) {
```

Comment:
This should be
```
_is_patched(false)
DEBUG_ONLY(COMMA _reads_is_archived(false)) {
```
Does the trailing COMMA version build?  (Either release or debug?)  I think it shouldn't, because
in either mode there is a extraneous trailing comma that I think the compiler should reject.

Reply:
No, it didn't build so I moved it to a higher spot in the initialization list. I wasn't familiar with this syntax but I think your recommendation is better.